apps/vaporgui/VRange.cpp

```diff
@@ -0,0 +1,81 @@
+#include "VRange.h"
+
+VRange::VRange( QWidget* parent, float min, float max, const std::string& minLabel,
+                const std::string& maxLabel )
+      : QWidget( parent )
+{
+    _layout    = new QVBoxLayout(this);
+
+    _minSlider = new VSlider( this, minLabel, min, max );
+    _maxSlider = new VSlider( this, maxLabel, min, max );
+    connect( _minSlider, SIGNAL( _valueChanged() ), this, SLOT( _respondMinSlider() ) );
+    connect( _maxSlider, SIGNAL( _valueChanged() ), this, SLOT( _respondMaxSlider() ) );
+
+    _layout->addWidget( _minSlider );
+    _layout->addWidget( _maxSlider );
+}
+
+VRange::~VRange() {}
```

Comment:
Rule of 0/5!! If a destructor is essentially empty, stick to 0.

Reply:
Thanks for the catch!

Reply:
This is an older file that's no longer needed.  Removed.